packages/js/generators.json

```diff
@@ -9,13 +9,6 @@
       "x-type": "library",
       "description": "Create a library"
     },
-    "application": {
```

Comment:
This is still referenced in the docs 🙃 
https://nx.dev/getting-started/nx-and-typescript#create-a-typescript-based-application

Tripped me up why this wasn't working.

Reply:
Propogated this message internally to see what we could do there. Thank you

Reply:
Tutorial broken ... that tutorial is simply the tutorial link that appear when one runs create-nx-workspace .. and it's broken ...